settings/LoanRules.js

```diff
@@ -162,7 +180,12 @@ class LoanRules extends React.Component {
   }
 
   render() {
-    if (!this.props.resources.loanTypes) {
+    const {
+      resources,
+      intl: { formatMessage },
+    } = this.props;
+
+    if (!resources.loanTypes) {
```

Comment:
Only loanTypes used from resources, so it could be movewd to destructuring above, like:
resources: { loanTypes }